proto/evm/enums.proto

```diff
@@ -16,4 +16,5 @@ enum PointerType {
 enum AssetType {
     TYPECW20 = 0;
```

Comment:
fwiw, the default value is always 0, so sometimes it can make sense to use `UNKNOWN` for this value to ensure someone intended to set it explicitly